src/main/java/org/apache/flink/graph/streaming/library/BipartitenessCheck.java

```diff
@@ -83,31 +89,17 @@ public Candidates foldEdges(Candidates candidates, Long v1, Long v2, NullValue e
 	}
 
 	/**
-	 * Reduce functions combine groups of elements to
-	 * a single value, by taking always two elements and combining them into one. Reduce functions
-	 * may be used on entire data sets, or on grouped data sets. In the latter case, each group is reduced
-	 * individually.
-	 * <p>
-	 * For a reduce functions that work on an entire group at the same time (such as the
-	 * MapReduce/Hadoop-style reduce), see {@link GroupReduceFunction}. In the general case,
-	 * ReduceFunctions are considered faster, because they allow the system to use more efficient
-	 * execution strategies.
-	 * <p>
-	 * The basic syntax for using a grouped ReduceFunction is as follows:
-	 * <pre>{@code
-	 * DataSet<X> input = ...;
-	 *
-	 * DataSet<X> result = input.groupBy(<key-definition>).reduce(new MyReduceFunction());
-	 * }</pre>
-	 * <p>
-	 * <p>
-	 * Like all functions, the ReduceFunction needs to be serializable, as defined in {@link java.io.Serializable}.
+	 * Implements the ReduceFunction Interface, applies reduce function to
+	 * combine group of elements into a single value.
```

Comment:
That's again quite general. What does this method compute in this particular algorithm?